geoportailv3/static/js/authentication/authenticationdirective.js

```diff
@@ -0,0 +1,227 @@
+goog.provide('app.authenticationDirective');
+
+goog.require('app');
+
+
+/**
+ * @param {string} appAuthenticationTemplateUrl URL to directive template.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+app.authenticationDirective = function(appAuthenticationTemplateUrl) {
+  return {
+    restrict: 'E',
+    controller: 'AppAuthenticationController',
+    controllerAs: 'userCtrl',
```

Comment:
The controller name is private to the isolate scope, so you can just use `ctrl` instead of `userCtrl` here. This is minor.